utils/src/main/java/software/amazon/awssdk/utils/NumericUtils.java

```diff
@@ -16,7 +16,9 @@
 package software.amazon.awssdk.utils;
 
 import software.amazon.awssdk.annotations.ReviewBeforeRelease;
+import software.amazon.awssdk.annotations.SdkProtectedApi;
 
+@SdkProtectedApi
```

Comment:
Public

Reply:
Before GA, sure. It's not ready to be public right now, though.